types/oauth2-server/index.d.ts

```diff
@@ -114,6 +114,10 @@ declare namespace OAuth2Server {
     }
 
     abstract class AbstractGrantType {
+        /**
+         * Model object
+         */
+        model: AuthorizationCodeModel | ClientCredentialsModel | RefreshTokenModel | PasswordModel | ExtensionModel;
```

Comment:
I don't understand why [model](https://github.com/oauthjs/node-oauth2-server/blob/30edbaac4ab65e548f7de533d6883f26e1f1728c/lib/grant-types/abstract-grant-type.js#L25) isn't part of [TokenOptions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a114dbb9225086dfeb33aab56bfd03b05e225c93/types/oauth2-server/index.d.ts#L216) along with the other [options](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a114dbb9225086dfeb33aab56bfd03b05e225c93/types/oauth2-server/index.d.ts#L220) like [accessTokenLifetime](https://github.com/oauthjs/node-oauth2-server/blob/30edbaac4ab65e548f7de533d6883f26e1f1728c/lib/grant-types/abstract-grant-type.js#L21). Could you shed some light? Thanks.